rmf_fleet_adapter/src/mock_traffic_light/main.cpp

```diff
@@ -54,11 +54,14 @@ rmf_fleet_msgs::msg::Location make_location(
   rclcpp::Time t,
   const Eigen::Vector3d& p,
   const std::string& map_name,
-  const std::size_t index)
+  const std::size_t index,
+  const std::optional<double>& speed_limit)
 {
   return rmf_fleet_msgs::build<rmf_fleet_msgs::msg::Location>()
```

Comment:
Maybe add a `const double approach_speed_limit = speed_limit.has_value() ? speed_limit.value() : 0.0;` which is used to assign the value below instead of `*speed_limit`

Reply:
Done https://github.com/open-rmf/rmf_ros2/pull/132/commits/510dcf83de2fbf34d6919cad7f2f91e1b23d429f